.github/ISSUE_TEMPLATE/INTERNAL.yml

```diff
@@ -0,0 +1,39 @@
+name: Team Internal
+description: Issue template used for tasks created by the NuGet team for themselves. Internal Use Only.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Preflight Checklist
+
+        **Do not file this issue until you've gone through this checklist.**
+
+        ### Basic Labels (required)
+
+        * Add **Regression** labels if this is a regression.
+        * Add a **Type:** label
+        * Add **Area:** and/or **Functionality:** labels
```

Comment:
Product probably deserves a mention here. 

Area is usually less important than Functionality and Product.